common-arcana.lic

```diff
@@ -236,4 +238,56 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def cast_spells(settings, spells)
+    infuse_om(!settings.osrel_no_harness, settings.osrel_amount)
+    spells.each do |name, data|
+      next if DRSpells.active_spells[name] && (data['recast'].nil? || DRSpells.active_spells[name].to_i > data['recast'])
+      while mana < 40
+        echo('Waiting on mana...')
+        pause 15
+      end
+
+      if data['ritual']
+        ritual(data)
+      else
+        cast_spell(settings, data)
+      end
+    end
+  end
+
+  def set_moon_data(data)
+    if data['moon']
+      check_moonwatch
+      # Look for a moon that will be up for the next few minutes
+      unless moon = UserVars.moons.find { |moon_name, moon_data| UserVars.moons['visible'].include?(moon_name) && moon_data['timer'] >= 4 }.first
+        echo "No moon available to cast #{data['abbrev']}"
+        return
+      end
+      data['cast'] = "cast #{moon}"
+    end
+    data
+  end
+
+  def check_moonwatch
+    unless Script.running? 'moonwatch'
+      echo 'moonwatch is not running. Starting it now'
+      custom_require.call('moonwatch')
+      echo "Run `;e autostart('moonwatch')` to avoid this in the future"
```

Comment:
Should `;` be `$clean_lich_char`?

https://github.com/rpherbig/dr-scripts/blob/master/setupaliases.lic#L11